exporters/elasticsearch/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ install(
 
 install(
   DIRECTORY include/opentelemetry/exporters/elasticsearch
-  DESTINATION include/opentelemetry/exporters/
+  DESTINATION include/opentelemetry/exporters
```

Comment:
Thanks for the fix, just wondering if this fails build in any scenario?

Reply:
This don't fail anything so far. I just observed there are path like `include/opentelemetry/exporters//etw/config.h` which may confuse.